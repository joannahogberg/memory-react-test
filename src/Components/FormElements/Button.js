import React from 'react';

function Button(props){
  return(
    <button className={ `m-2 ${ props.className }` } onClick={props.onClick}>
      { props.title }
    </button>
  )
}

export default Button;