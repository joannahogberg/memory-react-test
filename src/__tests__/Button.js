import React from 'react';
import toJSON from 'enzyme-to-json';
import { render } from 'enzyme';
import Button from '../Components/FormElements/Button.js';

it('Check button snapshot', () => {
    const wrapper = render(<Button />);
    expect(toJSON(wrapper))
    .toMatchSnapshot("Button snap");
})