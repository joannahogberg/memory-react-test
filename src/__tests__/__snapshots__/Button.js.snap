// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button snap 1`] = `
<button
  class="m-2 undefined"
>
  
</button>
`;
